api4/webhook.go

```diff
@@ -451,33 +452,39 @@ func incomingWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 	r.ParseForm()
 
 	var payload io.Reader
+	incomingWebhookPayload := &model.IncomingWebhookRequest{}
 	contentType := r.Header.Get("Content-Type")
 	if strings.Split(contentType, "; ")[0] == "application/x-www-form-urlencoded" {
 		payload = strings.NewReader(r.FormValue("payload"))
+	} else if strings.Split(contentType, "; ")[0] == "multipart/form-data" {
```

Comment:
Can you do `strings.HasPrefix(contentType, "multipart/form-data")` instead? That way we don't need to worry about the space not being there or a slightly different formatting, etc.

Reply:
should we do the same for `application/x-www-form-urlencoded`?

Reply:
done